kube-derive/src/custom_resource.rs

```diff
@@ -539,6 +539,115 @@ fn generate_hasspec(spec_ident: &Ident, root_ident: &Ident, kube_core: &Path) ->
     }
 }
 
+#[derive(FromAttributes)]
+#[darling(attributes(validated))]
+struct CELAttr {
+    rule: String,
+    message: Option<String>,
+}
+
+pub(crate) fn cel_validation(_: TokenStream, input: TokenStream) -> TokenStream {
```

Comment:
what is the first arg here for? the new proc macro seems take `args` and `input`, but you seem to be discarding `args`?

Reply:
Probably need to be checked, they are currently empty. Could be a good place for name overrides to avoid clashes